packages/text-input-react/src/TextField.tsx

```diff
@@ -37,16 +38,26 @@ export const TextField = ({
     variant,
     forceCompact,
     maxLength,
+    width,
     ...rest
 }: Props) => {
     const componentClassName = "jkl-text-field".concat(
         inline ? " jkl-text-field--inline" : "",
         forceCompact ? " jkl-text-field--compact" : "",
         className ? ` ${className}` : "",
     );
+    function getLength(): string | undefined {
+        if (width) return width; // prioritize width
```

Comment:
No no, this should be a lint-rule. It sort of looks pretty, but it degrades so fast, please add `{}`

Reply:
Haha, sorry 😅